src/picasso-def/components/not-fetched-pattern.js

```diff
@@ -11,7 +11,7 @@ export function getPattern(color) {
   dummyCanvas.height = ratio * 8;
   ctx.save();
   ctx.fillStyle = stroke.toString();
-  ctx.strokeStyle = stroke.toString();
+  ctx.strokeStyle = color === '#F2F2F2' ? '#F2F2F2' : '#ffffff';
```

Comment:
we should probably not just use === and instaed choose the color (of '#F2F2F2' or '#ffffff') that has highest contrast to stroke

Reply:
The case where we compare is for the default cases where #F2F2F2 is the value, rest all the cases its white as we had discussed- contrasting colors would lead to stripes in grey, black & corresponding dark colors and will not be a uniform pattern all over.